tools/CheckDates.scala

```diff
@@ -0,0 +1,110 @@
+import sys.process._
+
+object CheckDates {
+  val usage = """
+
+    This tool compares a copyright date in files with the date of last change in git.
+    If the message of last change contains "Documents date fixed" it will be ignored and 
```

Comment:
In the beginning I used
`git log --format=%s ./src/library/scala/runtime/ScalaNumberProxy.scala`
And split it via ("\\n"), but I found that some commits contain `\n` in the subject, and `messages.count` may not be equal to `dates.count` for one commit in this way.

So, I decided to parse entire body, and it's more precise now.